workflow/workflow_test.go

```diff
@@ -1213,6 +1213,124 @@ func TestInputCancelledStepWorkflow(t *testing.T) {
 	assert.Contains(t, err.Error(), "cannot construct any output")
 }
 
+var inputDisabledStepWorkflow = `
+version: v0.2.0
+input:
+  root: WorkflowInput
+  objects:
+    WorkflowInput:
+      id: WorkflowInput
+      properties:
+        step_enabled:
+          type:
+            type_id: bool
+steps:
+  simple_wait:
+    plugin:
+      src: "n/a"
+      deployment_type: "builtin"
+    step: wait
+    deploy:
+      deployer_name: "test-impl"
+      # Disabling steps waits for the value to be resolved before executing the step.
+      # So if it's working correctly, this should not cause any race conditions.
+      deploy_time: 0 # ms
```

Comment:
Do we have any means in place to detect the race condition?  If not, there's really no purpose to mentioning it here.

I think what you're trying to do with this comment is to explain why the `deploy_time` value is zero (or, why it is OK for it to be zero...), but I don't think you're succeeding.

It looks like we rarely specify `deploy_time` at all...do we need to specify it here?...if so, why?